config/config.exs

```diff
@@ -57,6 +57,8 @@ config :archethic, :mut_dir, "data"
 
 config :archethic, :marker, "-=%=-=%=-=%=-"
 
+config :archethic, :ownership_max_authorized_keys, 256
```

Comment:
I don't think there is need anymore for a limitation of ownership listing. The transaction's size policy will prevent any overflow.